src/border_agent/border_agent.cpp

```diff
@@ -496,8 +518,9 @@ void BorderAgent::PublishMeshCopService(void)
     // "xa" stands for Extended MAC Address (64-bit) of the Thread Interface of the Border Agent.
     txtList.emplace_back("xa", extAddr->m8, sizeof(extAddr->m8));
 
-    state       = GetStateBitmap(*instance);
-    stateUint32 = htobe32(state.ToUint32());
+    state                 = GetStateBitmap(*instance);
+    state.mEpskcSupported = GetEphemeralKeyEnabled();
```

Comment:
why not put this processing in `GetStateBitmap`?

Reply:
accept, thanks!